en/assembly/quick_start_pixhawk_mini.md

```diff
@@ -130,7 +133,11 @@ external safety button, as shown below.
 
 ## Motors
 
-![Pixhawk Mini - port for motors/servos](../../images/pixhawk_mini_port_main_out.png)
+The [Airframe Reference](airframe_reference.html) shows the default output port to motor/servos mappings for common airframe types.

```

Comment:
@dagar This is the fix to this [discuss comment](http://discuss.px4.io/t/pixhawk-mini-pwm-on-pin-3-is-a-constant-1500-us-resolved/3656/3). I decided to put it up in the top section rather than repeating in each sub-section. There is a chance that people will jump straight to a vehicle section and miss this, but I think very repetitive to have in each section.